src/citrine/informatics/design_spaces/hierarchical_design_space.py

```diff
@@ -179,6 +182,9 @@ def __init__(
     def _post_dump(self, data: dict) -> dict:
         data = super()._post_dump(data)
 
+        if self._settings:
+            data["settings"] = self._settings.dump()
```

Comment:
Note that for registration, they're added in a different place than they're read. This is intentional.